app/components/post_list/post/body/content/image_preview/index.tsx

```diff
@@ -75,14 +74,6 @@ const ImagePreview = ({expandedLink, isReplyPost, link, metadata, postId, theme}
         }
     }, [link]);
 
-    useDidUpdate(() => {
-        if (expandedLink) {
-            setImageUrl(expandedLink);
-        } else if (link !== imageUrl) {
-            setImageUrl(link);
-        }
-    }, [link]);
-
```

Comment:
Oh I messed it up, sorry for that,. on my first review I misread this and thought it was `useWhyDidYouUpdate` hehehe, can you bring this back?

Again, sorry about that